learn/experimental/metrics.md

```diff
@@ -0,0 +1,11 @@
+# Metrics
+
+The `/metrics` endpoint exposes data compatible with [Prometheus](https://prometheus.io/). You can monitor this endpoint to gain valuable insight into Meilisearch's behavior and performance. This information can then be used to optimize an application's performance, troubleshoot issues, or improve the overall reliability of a system.
+
+[Visit the GitHub discussion to learn how to use this feature](https://github.com/meilisearch/product/discussions/625).
+
+Are you using this feature? Meilisearch wants to hear from you! Drop us a few lines telling about your experience in the [GitHub discussion](https://github.com/meilisearch/product/discussions/625). All feedback is useful and helps make Meilisearch better and easier-to-use.
+
+::: warning
+`/metrics` is an experimental feature. Experimental features are unstable and might be incompatible between releases. Meilisearch does not guarantee an experimental feature will become stable.
```

Comment:
It's a detail, you can disagree, but I would like to make this more precise regarding the "unstable" word. It's unstable regarding the usage, not unstable because buggy. I don't want to afraid the users 😊 
When I read the sentence "Experimental features are unstable and might be incompatible between releases" -> I understand "this is buggy" AND "this can be incompatible btw releases". The truth is "it's unstable BECAUSE might be incompatible btw releases"

Reply:
```suggestion
`/metrics` is an experimental feature. Experimental features are unstable because might be incompatible between releases.
Also, Meilisearch does not guarantee an experimental feature will become stable.
```

Reply:
My changes are maybe not the right one, but the points are
- clarify the "unstable" word
- separate both points, since these are 2 distinct ideas. I don't want the user to miss the second point.

Reply:
Got it, will fix this and request a new review in a bit